experimental/package_crawler_static.py

```diff
@@ -0,0 +1,139 @@
+import ast
```

Comment:
You are checking this in so you can work on something else, not because you want a thorough review, right?

Reply:
In any case, you need a license at the top of every file.

Reply:
That's true, but I think if you're seeing something egregious in this commit now, it might be better to point it out now while everything is fresh.